resources/zib/zib-CareTeam.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-CareTeam" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-CareTeam" />
+  <name value="ZibCareTeam" />
+  <title value="zib CareTeam" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A care team is a group of health professionals and other people involved in caring for a patient for one or more problems." />
+  <purpose value="This CareTeam resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) CareTeam v1.0(2020)](https://zibs.nl/wiki/CareTeam-v1.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-careteam-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/CareTeam-v1.0(2020EN)" />
+    <name value="zib CareTeam-v1.0(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-healthprofessional-v3.5-2020EN" />
+    <uri value="https://zibs.nl/wiki/HealthProfessional-v3.5(2020EN)" />
+    <name value="zib HealthProfessional-v3.5(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="CareTeam" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CareTeam">
+      <path value="CareTeam" />
+      <short value="CareTeam" />
+      <definition value="Root concept of the CareTeam information model.This root concept contains all data elements of the CareTeam information model." />
+      <alias value="ZorgTeam" />
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.1" />
+        <comment value="CareTeam" />
+      </mapping>
+    </element>
+    <element id="CareTeam.name">
+      <path value="CareTeam.name" />
+      <short value="CareTeamNaam" />
+      <definition value="The name of the care team." />
+      <alias value="ZorgTeamNaam" />
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.12" />
+        <comment value="CareTeamNaam" />

```

Comment:
I think this is a mistake in the zib this must be "CareTeamName"

Reply:
Yeah, the zib makes a mistake here. Fixed it.